ReleaseNotes.md

```diff
@@ -1,14 +1,23 @@
 # Release Notes&mdash;NJOY21
 Given here are some release notes for NJOY21. Each release is made through a formal [Pull Request](https://github.com/njoy/NJOY21/pulls) made on GitHub. There are links in this document that point to each of those Pull Requests, where you can see in great details the changes that were made. Often the Pull Requests are made in response to an [issue](https://github.com/njoy/NJOY21/issues). In such cases, links to those issues are also given.
 
-## [NJOY21 1.0.5](https://github.com/njoy/NJOY21/pull/99)
+## [NJOY21 1.0.7](https://github.com/njoy/NJOY21/pull/101)
+This update fixes a bug in the error checking for ACER. This bug was fixed in lipservice:
+
+- [ACER NGEND=0](https://github.com/njoy/lipservice/pull/14) which resolves [Issue #15](https://github.com/njoy/lipservice/issues/15)
+
+## [NJOY21 1.0.6](https://github.com/njoy/NJOY21/pull/99)
```

Comment:
I assume that the release notes were previously incorrect in lacking the correct label for 1.0.6 and lacking a description of 1.0.5.

Reply:
Your assumption is correct. There was a "bug" in the incorrect label of 1.0.6. Some of the release notes were simply missing for some of the older ones.